recipes/pycoal/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "pycoal" %}
+{% set version = "0.5.1" %}
+{% set sha256 = "d066af5e92d1d8250d972c7a683f70403df40e2aa3bac2d45655b301df7c7a18" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - numpy
+    - spectral
+
+test:
+  imports:
+    - pycoal
+
+about:
+  home: https://github.com/capstone-coal/pycoal/
+  license: Apache 2.0
+  license_family: Apache
+  license_file: LICENSE
```

Comment:
@browtayl Thanks for the explanation. FYI, I had a separate issue arise recently that may prompt a migration to GPL3, though it will likely be at least a few more weeks for that to be resolved. If it does happen, I'll post a comment here to let you know.